stylelint.config.js

```diff
@@ -44,6 +44,12 @@ module.exports = {
 		],
 		'order/properties-alphabetical-order': true,
 		'selector-class-pattern': '-?ds-[a-z][a-zA-Z_]*',
+		'unit-no-unknown': [
+			true,
+			{
+				ignoreUnits: ['lh'],
```

Comment:
is it ok, that we ignore it globally? Maybe it's better to disable error locally, so we can put a fallback there?